clients/java/src/main/java/io/zeebe/client/impl/util/SocketUtil.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.zeebe.client.impl.util;
+
+import java.net.Inet6Address;
+import java.net.InetSocketAddress;
+
+public final class SocketUtil {
```

Comment:
Can we add a comment explaining why it's duplicated? i.e. `utils/` is not a Java 8 module so we can't directly depend from it, hence the duplication. And possibly if it got more complex we'd need to figure out how to share it :sweat_smile: